src/app/app.component.html

```diff
@@ -22,7 +22,23 @@
     </mat-form-field>
   </div>
 
-  <div class="dataDiv">
-    <app-weather-data *ngIf=" locationID || locationID == 0 " [data]="weatherData[locationID]"></app-weather-data>
+  <div class="centerPage">
+    <ng-container *ngIf=" locationID || locationID == 0">
+      <div class="timeBtns">
+        <b>Select a Time</b>
+        <ng-container *ngFor=" let time of weatherData[locationID].timeSeries; index as i ">
+          <div *ngIf="i<=24">
+            <ng-container
+              *ngIf="(weatherData[locationID].timeSeries[0].time | date:'d') == (weatherData[locationID].timeSeries[i].time | date:'d')">
```

Comment:
is this checking that the time is on the same day? do you need to do this?

the ngIf could also go directly on the button rather than needing an ng-container

Reply:
It is making only all hours of the current day get displayed, on the spec it it implys that they will be checking on the day before they wanna leave so i left it with same day. i couldnt figure out/decide how to display all the other days without it getting too messy, so for now current day is displayed. potentially a future feature includes a day selection/display?